help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,245 @@
+
+### New Account
+
+To define a new Merkury Managed S3 location:
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image4.png){width="6.5in"
+height="3.509027777777778in"}
+
+### Existing Account
+
+Accounts already defined using the Merkury Enterprise Connections
+destination appear in a list pop-up. When selected, you can see details
+on the account in the right rail. View the example from the UI, when you
+navigate to **Destinations** \> **Accounts**;
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image5.png){width="7.260470253718285in"
+height="1.5785312773403324in"}
+
+### **Fill in destination details
+
+To configure details for the destination, fill in the required and
+optional fields below. An asterisk next to a field in the UI indicates
+that the field is required.
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image6.png){width="6.5in"
+height="3.877083333333333in"}
+
+-   **Name (Required)** - The name the destination will be saved under
+
+-   **Description** - Short explanation of the destination's purpose
+
+-   **Bucket Name (Required)** - Name of the Amazon S3 bucket set up on
+    S3
+
+-   **Folder Path (Required)** - If subdirectories in a bucket are used
+    a path must be defined, or '/' to reference the root path.
+
+-   **File Type** - Select the format Experience Platform should use for
+    the exported files. Currently, the only file type Acxiom processing
+    will be expecting is CSV
+
+![A white background with black text Description automatically
+generated](./merkury-connections/media/image7.png){width="6.5in"
+height="1.336111111111111in"}
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image8.png){width="4.875in"
+height="7.791666666666667in"}
+
+## **Enable alerts
+
+You can enable alerts to receive notifications on the status of the
+dataflow to your destination. Select an alert from the list to subscribe
+to receive notifications on the status of your dataflow. For more
+information on alerts, see the guide on [subscribing to destinations
+alerts using the
+UI](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/alerts).
+
+When you are finished providing details for your destination connection,
+select **Next**.
+
+## Activate audiences to this destination
+
+![A screenshot of a computer screen Description automatically
+generated](./merkury-connections/media/image9.png){width="7.04362532808399in"
+height="2.709839238845144in"}
+
+Read [Activate audience data to batch profile export
+destinations](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations) for
+instructions on activating audiences to this destination.
+
+## Mapping suggestions
```

Comment:
I noticed this section is copied from the Acxiom page, do the same mapping suggestions apply to the Merkury destination? 

https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement#mapping-suggestions

Reply:
Addressed